kriswallsmith/buzz/0.17/config/packages/buzz.yaml

```diff
@@ -0,0 +1,30 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
+    _defaults:
+        public: false
+
+    # Clients
+    Buzz\Client\Curl:
+        arguments:
+            - timeout: 10
+
+    Buzz\Client\FileGetContents:
+        arguments:
+            - timeout: 10
+
+    Buzz\Client\MultiCurl:
+        arguments:
+            - timeout: 10
+
+    # Browser
+    Buzz\Browser:
+        arguments: ['@Buzz\Client\Curl']
+        calls:
+            - ['addMiddleware', ['@Buzz\Middleware\ContentLengthMiddleware']]
+            - ['addMiddleware', ['@Buzz\Middleware\LoggerMiddleware']]
+
+    # Middlewares
+    Buzz\Middleware\ContentLengthMiddleware: ~
+
+    Buzz\Middleware\LoggerMiddleware:
+        arguments: ['@?logger']
```

Comment:
what about adding the `monolog.logger` tag to log in a dedicated channel (`buzz` looks like a sensible default) instead of the default `app` one ? It will not hurt when using the fallback logger rather than MonologBundle.

Reply:
So many great suggestions. Of course we should!